### PR TITLE
[#30] Rename problems to conditions

### PR DESCRIPTION
**Pull request by vitorbaptista**
opentrials/opentrials#30

### CODE REVIEW

File: api/models/condition.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+const bookshelf = require('../../config').bookshelf;
+const BaseModel = require('./base');
+
+const Condition = BaseModel.extend({
+  tableName: 'conditions',
+  visible: [
+    'id',
+    'name',
+    'type',
```
Comment (vitorbaptista) [inline api/models/condition.js]: `type` now is just `condition` or `other`, as far as I know. Maybe we should also remove this attribute? /cc @roll

Comment (roll) [inline api/models/condition.js]: of course remove :+1:

**Comment by roll** [comment]
:+1:

File: api/models/condition.js (hunk 1 above)
Comment (vitorbaptista) [inline api/models/condition.js]: Done :+1:

**Pull request merged**